cmd/cluster/support/post.go

```diff
@@ -79,34 +79,24 @@ The cluster has a second failing ingress controller, which is not supported and
 		Args:              cobra.NoArgs,
 		DisableAutoGenTag: true,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			if err := p.Run(p.ClusterID); err != nil {
-				return fmt.Errorf("error posting limited support reason: %w", err)
-			}
-			return nil
+			return p.Run(p.ClusterID)
 		},
 	}
 
 	// Define required flags
-	postCmd.Flags().StringVarP(&p.ClusterID, "cluster-id", "c", "", "Intenal Cluster ID (required)")
+	postCmd.Flags().StringVarP(&p.ClusterID, "cluster-id", "c", "", "Internal Cluster ID (required)")
 	postCmd.Flags().StringVarP(&p.Template, "template", "t", "", "Message template file or URL")
 	postCmd.Flags().StringArrayVarP(&p.TemplateParams, "param", "p", p.TemplateParams, "Specify a key-value pair (eg. -p FOO=BAR) to set/override a parameter value in the template.")
 	postCmd.Flags().Var(&p.Misconfiguration, MisconfigurationFlag, "The type of misconfiguration responsible for the cluster being placed into limited support. Valid values are `cloud` or `cluster`.")
 	postCmd.Flags().StringVar(&p.Problem, ProblemFlag, "", "Complete sentence(s) describing the problem responsible for the cluster being placed into limited support. Will form the limited support message with the contents of --resolution appended")
 	postCmd.Flags().StringVar(&p.Resolution, ResolutionFlag, "", "Complete sentence(s) describing the steps for the customer to take to resolve the issue and move out of limited support. Will form the limited support message with the contents of --problem prepended")
 	postCmd.Flags().StringVar(&p.Evidence, EvidenceFlag, "", "(optional) The reasoning that led to the decision to place the cluster in limited support. Can also be a link to a Jira case. Used for internal service log only.")
 
-	// Mark cluster-id as required
-	postCmd.MarkFlagRequired("cluster-id")
+	_ = postCmd.MarkFlagRequired("cluster-id")
 
 	return postCmd
 }
 
-func (p *Post) Init() error {
-	userParameterNames = []string{}
-	userParameterValues = []string{}
```

Comment:
These are still global variables and need to be factored better but moved this into `setup()`.